### PR TITLE
feat: show server timings on search website

### PR DESCRIPTION
**Pull request by drew-harris**

### CODE REVIEW

**Comment by drew-harris** [comment]
image

File: frontends/search/src/components/ResultsPage.tsx [hunk 1]
```diff
@@ -191,6 +211,44 @@ const ResultsPage = (props: ResultsPageProps) => {
...
+          <label class="flex items-center space-x-2">
+            <span class="text-sm font-medium text-gray-900 dark:text-white">
+              Show Server Timings
+            </span>
+            <div
+              class="text-primary-600 focus:ring-primary-500 h-3 w-3 rounded border-gray-300"
+              // checked={showServerTimings()}
+              // onChange={(e) => {
+              //   setShowServerTimings(e.target.checked);
+              // }}
```
Comment (cdxker) [inline frontends/search/src/components/ResultsPage.tsx:232]:
```suggestion
```
remove comment

**Review by cdxker — CHANGES_REQUESTED** [review]
Looks great! just one small thing then it'll be good to merge.

**Review by cdxker — APPROVED** [review]
Great!

**Pull request merged**